docs/learning-powershell/README.md

```diff
@@ -119,9 +117,7 @@ Note that all bash commands should continue working on PowerShell session.
 [getstarted-with-powershell]: https://channel9.msdn.com/Series/GetStartedPowerShell3
 [why-learn-powershell]: https://blogs.technet.microsoft.com/heyscriptingguy/2014/10/18/weekend-scripter-why-learn-powershell/
 [Using Windows PowerShell for Administration]: https://docs.microsoft.com/powershell/scripting/getting-started/fundamental/using-windows-powershell-for-administration?view=powershell-6
-[ebook-from-powershell.com]: http://powershell.com/cs/blogs/ebookv2/default.aspx
```

Comment:
I found this here http://community.idera.com/database-tools/powershell/powertips/b/ebookv2

Reply:
Perfect, will update.